### PR TITLE
Updated typos and crate version in README.md

### PR DESCRIPTION
**Pull request by michalusio**
Fixed a few typos and updated the sample code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Web application.
 * Good flexibility and extensibility of materials with node-based material system
 * Web application compatible by compiling to WebAssembly
 
-## Documents
+## Documentation
```
Comment (takahirox) [inline README.md:33]: English question. How Documents and Documentation are different?

Comment (michalusio) [reply README.md:33]: Documents are a multiple of document.
Documentation is a set of materials describing how to use, and the workings of a system, library or an API.

Documentation is a technical term, while documents are an office term.

In other words - all documentation are document(s), but not all documents are documentation.

Comment (takahirox) [reply README.md:33]: Thanks!

**Comment by takahirox** [comment]
Looking good to me, thanks!

**Pull request merged**